department-of-veterans-affairs/caseflow
# Mkerns/caseflow 1384 ihp task cancellation

Resolves [CASEFLOW-1384](https://vajira.max.gov/browse/CASEFLOW-1384)

### Description
Added logic to cancel and update the IhpTask when a new POA is updated.

### Acceptance Criteria
- [x] Active IHP task with the former POA is cancelled
- [x] New IHP task is created and assigned to the new POA on the case

### Testing Plan
1. Go to Case Details page
2. Click `POA Refresh` button
2. Scroll to Caseflow Timeline Section
3. Verify that the `IhpTask` cancellation shows up in the Timeline
